<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BarchartConditionInput</class>
 <widget class="QWidget" name="BarchartConditionInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="color_toolButton">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="label_lineEdit">
     <property name="minimumSize">
      <size>
       <width>175</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>type condition label here</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="condition_lineEdit">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>type your condition here (ex: y &lt; yHalf)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="eliminateSpikes_checkBox">
     <property name="text">
      <string>Hide spikes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="remove_toolButton">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="add_toolButton">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
